styles/components/button.scss

```diff
@@ -131,3 +187,33 @@
   //   border: 1px solid $gray;
   // }
 }
+
+.btn-group {
+  button,.btn + .btn{
+    margin-right: 0px;
+    margin-left: 0px;
+  }
+  button,.btn {
+    border-radius: 0px;
+    border: 1px solid #E7E7E7;
+  }
+  button:first-child {
+    border-top-left-radius: $border-radius-base2x;
+    border-bottom-left-radius: $border-radius-base2x;
+  }
+  button:last-child {
+    border-top-right-radius: $border-radius-base2x;
+    border-bottom-right-radius: $border-radius-base2x;
+  }
+
+}
+
+.-text-left {
```

Comment:
I don't think these belong nor do we need these for the button. If we want to center the text, let's add padding to the buttons, not width.